index.html

```diff
@@ -86,6 +86,192 @@ <h2 class="description">I’m a Software Developer</h2>
             </li>
         </ul>
     </section>
+    <ul class="works">
```

Comment:
- [ ] The container of the section is a `section` tag. Replace `ul` with `section`. We are using section because this is basically a container and not list of items.

Reply:
Thank you for the review, 

I think I have a serious issue though.

If you have a look at screenshot attached, you'll find that last week I was asked to change the section to 'ul' and 'li'. Today you are asking me to change it back. I don't mind doing it, but how do I know which one is the best way and how to I make sure that somebody else doesn't get back to me asking me to change it back to  'ul' and 'li'.


![191679234_483789212711723_145053281348025917_n](https://user-images.githubusercontent.com/70698085/121525966-fd698700-ca09-11eb-9177-0379c9b95ccc.png)

Reply:
No worries.
Thanks for the clarification. I'm going to work on it right now!

Reply:
Changes made:
- Button size adjusted
- Images folder created for the images
- Section tags added for the cards